submissions/ifeanyiLawson/css/vendorspage.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vendors Page</title>
+    <link rel="stylesheet" href="style2.css">
+</head>
+<body>
+    <header id="btn">
+    <nav>
+       <a href="/">Food<span class="bag">Bag</span></a>
```

Comment:
can you use a p or h1 tag before the a href tag, this might solve the filing test error

Reply:
I have done that, I got the same result.